module/Application/src/View/Helper/Breadcrumbs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Laminas\View\Helper\Placeholder\Container\{
+    AbstractContainer,
+    AbstractStandalone,
+};
+
+/**
+ * Helper for setting and retrieving breadcrumbs.
+ */
+class Breadcrumbs extends AbstractStandalone
+{
+    public function __construct()
```

Comment:
Je moet hier ook even een constructor call naar de parent toevoegen

Reply:
Hoezo moet de constructor trouwens uberhaubt iets returnen?

Reply:
> 
> 
> Hoezo moet de constructor trouwens uberhaubt iets returnen?

Dat is de magie van de Placeholder viewhelper (waar dit eigenlijk op gebaseerd is), het idee is dat je dingen kan laden door

```php
<?php

$this->placeholder()->method($content);
```

en wanneer je het nodig hebt doe je (wat dus de `__toString()` magic method aanroept)

```php
<?php

echo $this->placeholder()
```

Het ziet er misschien een beetje raar uit, maar het is precies hetzelfde als de `headTitle`, `headScript`, etc., die we al gebruiken.